config/alertmanager/templates/slack-alert-text.tmpl

```diff
@@ -1,4 +1,19 @@
-{{ define "slack-alert-text" }}
-{{ range .Alerts }}* {{ if eq .Status "firing" }}{{ .Annotations.error }}{{ else }}~{{ .Annotations.error }}~{{ end }}
-{{ end }}
-{{ end }}
\ No newline at end of file
+{{ define "__alert_silence_link" -}}
```

Comment:
How did you design and test these templates? Or even where is the reference for what can be used?

Reply:
This is the reference:
https://prometheus.io/docs/alerting/latest/notifications/
https://prometheus.io/docs/alerting/latest/notification_examples/

For testing you can use `./run --dev`, which will run the whole stack (prometheus/alertmanager/exporters/...) but push the alerts to the `#monitor-test` instead. I added some dummy always-failing alerts to get some data on that channel.

This was mostly inspired by https://gist.github.com/milesbxf/e2744fc90e9c41b47aa47925f8ff6512 and https://hodovi.cc/blog/creating-awesome-alertmanager-templates-for-slack/